### PR TITLE
Lint models/repo_mirror.go

### PR DESCRIPTION
**Pull request by Bwko**

### CODE REVIEW

File: models/repo_mirror.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"code.gitea.io/gitea/modules/sync"
 )
 
+//MirrorQueue holds an UniqueQueue object of the mirror
```
Comment (lunny) [inline models/repo_mirror.go]: space

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by strk** [comment]
LGTM

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**